spec/time_for_a_boolean_spec.rb

```diff
@@ -81,7 +82,7 @@
 
       it 'is false if the attribute date is in the future' do
         klass.time_for_a_boolean :attribute, :attribute_on
-        allow(object).to receive(:attribute_on).and_return(Date.current + 1)
+        allow(object).to receive(:attribute_on).and_return(Time.current + 1)
```

Comment:
Mmm good point-- that would recalculate the current time whenever it's called right?